test/src/main/java/org/apache/accumulo/harness/AccumuloITBase.java

```diff
@@ -90,7 +91,9 @@ public static File createTestDir(String name) {
     if (name == null) {
       return baseDir;
     }
-    File testDir = baseDir.toPath().resolve(name).toFile();
+    String uniqueName = String.format("%s-%d-%d", name, System.currentTimeMillis(),
+        RANDOM.get().nextInt(Short.MAX_VALUE));
+    File testDir = baseDir.toPath().resolve(uniqueName).toFile();
```

Comment:
There may be a few ITs that extend MiniClusterHarness or AccumuloClusterHarness directly, rather than extend AccumuloITBase or SharedMiniClusterBase. Those test dir names would be in AccumuloClusterHarness.createTestDir.